.travis.yml

```diff
@@ -47,7 +47,7 @@ env:
     - MYSQL_PORT=3306
     - DIST_PACKAGE_TARGET=DEB
     - SKIP_TEST_ON_ERROR=0
-    - HOMEBREW_NO_AUTO_UPDATE=1
+    - HOMEBREW_NO_AUTO_UPDATE=0
```

Comment:
Negatively named boolean variables take so much longer to understand. This would've been much simpler if it was:
```
- HOMEBREW_AUTO_UPDATE=1
```
But anyway, that's not really a concern of this PR.